### PR TITLE
404 Lists, again

### PR DESCRIPTION
**Pull request by bogachenko**
@collinbarrett I have a question, that these links need to be deleted?

https://ransomwaretracker.abuse.ch/downloads/CW_PS_IPBL.txt
https://ransomwaretracker.abuse.ch/downloads/CW_PS_DOMBL.txt
https://raw.githubusercontent.com/eEIi0A5L/adblock_filter/master/beta.txt

### CODE REVIEW

**Comment by bogachenko** [comment]
Блять

**Comment by bogachenko** [comment]
о господь всемогущий

**Comment by bogachenko** [comment]
what's wrong with ID?

File: data/FilterList.json [hunk 1]
```diff
@@ -8969,8 +8919,7 @@
     "licenseId": 2,
     "name": "Magno's Adblock List",
     "syntaxId": 3,
-    "viewUrl": "https://raw.githubusercontent.com/magnobiet/adblock-plus-list/gh-pages/list.txt",
-    "viewUrlMirror1": "https://adp.magnobiet.com/list.txt"
+    "viewUrl": "https://adp.magnobiet.com/list.txt",
```
Comment (collinbarrett) [inline data/FilterList.json]: maybe this extra trailing comma? I'm not sure. something is goofy with the Travis build which used to help isolate errors.

**Comment by collinbarrett** [comment]
Ah. See [here](https://travis-ci.org/collinbarrett/FilterLists/jobs/492500628#L630). Check the Forks.json to see if there is a record there that needs removing.

Thanks.

**Comment by bogachenko** [comment]
I fixed, I'm waiting ...

answer please

> 
> 
> @collinbarrett I have a question, that these links need to be deleted?
> 
> https://ransomwaretracker.abuse.ch/downloads/CW_PS_IPBL.txt
> https://ransomwaretracker.abuse.ch/downloads/CW_PS_DOMBL.txt
> https://raw.githubusercontent.com/eEIi0A5L/adblock_filter/master/beta.txt

**Comment by bogachenko** [comment]
Coooooooollin, answer please, they need to be deleted or not, what do you think?

**Comment by collinbarrett** [comment]
Oh, haha, sorry. Yeah, if they are empty, we can go ahead and delete them.

**Pull request merged**